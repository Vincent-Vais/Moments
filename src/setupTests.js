import React from "react";
import { configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import configureStore from "redux-mock-store";
import { createMuiTheme } from "@material-ui/core";
import { ThemeProvider } from "@material-ui/core/styles";

configure({ adapter: new Adapter() });

export function MockTheme({ children }) {
  const theme = createMuiTheme({});
  return <ThemeProvider theme={theme}>{children}</ThemeProvider>;
}

const middlewares = [];
export const mockStore = configureStore(middlewares);

export const mockClasses = {};
