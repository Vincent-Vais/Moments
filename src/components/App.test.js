import React from "react";
import { shallow } from "enzyme";
import App from "./App.component";
import { mockClasses } from "../setupTests";

const getPresentComponent = () => <App classes={mockClasses} />;

describe("App component", () => {
  it("renders presentational component", () => {
    const component = shallow(getPresentComponent());
    expect(component).toMatchSnapshot();
  });
});
