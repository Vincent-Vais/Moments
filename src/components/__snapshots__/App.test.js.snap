// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders presentational component 1`] = `
<div>
  <Component />
  <Component />
</div>
`;
